### PR TITLE
Adding class page and linking it to class page with tabs. Adding mess…

### PR DESCRIPTION
**Pull request by SanaJallouli**
Screen Shot 2021-10-15 at 4 21 54 PM

Screen Shot 2021-10-15 at 4 22 04 PM
Screen Shot 2021-10-15 at 4 22 09 PM

Screen Shot 2021-10-15 at 4 22 14 PM

-Add class tabs 

-Create stack to navigate from classes to a specific class. dummy button added in classes to access the class tabs. Back button to go from class tabs to the list of classes. 

-Create stack between messages page and when to meet page. Dummy button to go from messages to when to meet.  Back button to go back from when2meet.

### CODE REVIEW

File: src/screens/Classes/ClassTabs.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+import React , { Component } from "react";
```
Comment (nuzurie) [inline src/screens/Classes/ClassTabs.js:2]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8258429)
```suggestion
import React , { Component } from 'react';
```

File: src/Messages/Messages.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import React , { Component } from "react";
```
Comment (nuzurie) [inline src/Messages/Messages.js:1]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8258429)
```suggestion
import React , { Component } from 'react';
```

File: src/Messages/Messages.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        component={When2meet}
+  
+      />
+    </Stack2.Navigator>
+
+    )
+  }
+}
+
+export default Messages;
```
Comment (stellakimnguyen) [inline src/Messages/Messages.js]: Add empty line at end of file! (It doesn't change the workings of the app, but helps the reading from input stream to know it has finished reading the file)

File: src/Messages/Messages.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  StyleSheet,
+  View,
+  Text,
+  Pressable,
+} from 'react-native';
+import When2meet from './When2meet'
+
+
+
+const Stack2 = createStackNavigator();
```
Comment (stellakimnguyen) [inline src/Messages/Messages.js]: Potentially change const name to something more relevant (ClassStack?)

File: src/screens/Classes/ClassTabs.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  
+    return (
+      
+        <Tab.Navigator
+        
+        // style to be updated later 
+           screenOptions={{
+                tabBarLabelStyle: { fontSize: 10, fontWeight:'bold',},
+                tabBarStyle:{paddingTop: 10,  // just to visualize before adding the view containing the screen title
```
Comment (stellakimnguyen) [inline src/screens/Classes/ClassTabs.js:31]: According to the comment, is this still needed? :)

File: src/screens/Classes/Classes.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+      />
+    </Stack.Navigator>
+
+)
+}
+}
+
+
+
+export default Classes;
```
Comment (stellakimnguyen) [inline src/screens/Classes/Classes.js]: Add empty line!

File: src/screens/Feed.js [hunk 7]
```diff
@@ -2,10 +2,6 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import styled from 'styled-components';
 import { StyleSheet, Text, View } from 'react-native';
-import theme from '../styles/theme.style.js';
```
Comment (stellakimnguyen) [inline src/screens/Feed.js:5]: Keep the theme here just in case we need it for later :)

File: src/screens/Profile.js [hunk 8]
```diff
@@ -2,7 +2,6 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import styled from 'styled-components';
 import { StyleSheet, Text, View } from 'react-native';
-import theme from '../styles/theme.style.js';
```
Comment (stellakimnguyen) [inline src/screens/Profile.js:5]: Same thing here

File: src/screens/Schedule.js [hunk 9]
```diff
@@ -2,7 +2,6 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import styled from 'styled-components';
 import { StyleSheet, Text, View } from 'react-native';
-import theme from '../styles/theme.style.js';
```
Comment (stellakimnguyen) [inline src/screens/Schedule.js:5]: and here

**Review by stellakimnguyen — APPROVED** [review]
Once you review the feedback and fix the merge conflicts, consider it approved!! 🎉🥂

**Pull request closed without merging**